apps/web/src/components/settings/ProfileSettings.tsx

```diff
@@ -77,8 +106,10 @@ export default function ProfileSettings({
 					</div>
 					<Button
 						onClick={async () => {
```

Comment:
nit: I do not think this needs to be async.

Reply:
It does for line 121: `await encodeFileAsBase64(newProfileImage);`
I haven't dug into this function to determine if it needs to be async as I didn't right it, but it's something I'll look at later